src/core/hiwire.c

```diff
@@ -60,6 +65,14 @@ EM_JS_NUM(int, hiwire_init, (), {
     let idval = _hiwire.counter[0];
     _hiwire.objects.set(idval, jsval);
     _hiwire.counter[0] += 2;
+#ifdef DEBUG_F
```

Comment:
Could you add a section about building core with debug flags in https://pyodide.org/en/latest/development/building-from-sources.html as currently it's not really documented..

Reply:
I think I should also change `make debug` not to set `-s ASSERTIONS=2`, I find that it slows compilation and runtime down a lot and makes a lot of noise which isn't very useful. On the other hand, `DEBUG_F` usually provides really helpful diagnostics. I've been compiling with `EXTRA_CFLAGS="-D DEBUG_F" make minimal` to avoid the `ASSERTIONS`.